utility/auth.js

```diff
@@ -35,6 +35,14 @@ const Auth = ({ children }) => {
     return () => unsubscribe();
   }, []);
 
+  // useEffect(() => {
+  //   return firebase.auth().onAuthStateChanged(() => {
+  //     setUser(user);
+  //     console.log(router.pathname);
+  //     router.push(router.pathname);
+  //   });
+  // }, [router.pathname]);
+
```

Comment:
Not really sure what this does, so if someone has some insight 👀 

For context this is the corresponding code in assessment-portal:
```
import { useHistory } from 'react-router-dom'
const history = useHistory()
...
  useEffect(() => {
    return app.auth().onAuthStateChanged(user => {
      setUser(user)
      history.push('/Assessment')
    })
  }, [history])
...
```

It looks like we're checking auth and pushing `/Assessment` onto the navigation stack (or whatever you call it, don't have too much experience with routers), but that's pretty much the only page. Don't think we need this unless I'm misunderstanding (which I very well could be).